core/src/main/res/xml/dashboard_contributors.xml

```diff
@@ -16,7 +16,7 @@
     <contributor
         name="Technologx"
         contribution=""
-        image="https://avatars0.githubusercontent.com/u/17693282?v=3&amp;s=400"
-        link="https://github.com/Technologx" />
+        image="https://lh3.googleusercontent.com/xzdaNykgAewl8vNKG-CIxeBGWUnr02NV-KBTHRKBCiGD_5ydpBzBGRMep5Nu5MO3G7wcvCFjYLUzEw=w1280-h800-no"

```

Comment:
Reduce the resolution, something like https://lh3.googleusercontent.com/xzdaNykgAewl8vNKG-CIxeBGWUnr02NV-KBTHRKBCiGD_5ydpBzBGRMep5Nu5MO3G7wcvCFjYLUzEw=w300-no